pmd-core/src/main/java/net/sourceforge/pmd/processor/PmdRunnable.java

```diff
@@ -130,7 +128,7 @@ private void processSource(String sourceCode,
             languageVersion,
             filename,
             sourceCode,
-            SemanticErrorReporter.noop() // TODO
+            SemanticErrorReporter.reportToLogger(LOGGER)
```

Comment:
I think, we'll need to use the new constructor add provide here the auxclasspath (not sure from where to get this here....)

Reply:
Ah right, thanks for finding this :) The configuration has the classpath